src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Telemetry/ITelemetryReporterInitializer.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.Telemetry;
+
+internal interface ITelemetryReporterInitializer
```

Comment:
ℹ️ This interface is split off from `ITelemetryReporter`, since `DevKitTelemetryReporter` is the only implementation that needs `InitializeSession`.